src/Sylius/Bundle/CoreBundle/Controller/OrderController.php

```diff
@@ -111,4 +121,173 @@ public function historyAction(Request $request)
 
         return $this->viewHandler->handle($configuration, $view);
     }
+
+    /**
+     * @param Request $request
+     * @param int $orderId
+     *
+     * @return Response
+     */
+    public function payAction(Request $request, $orderId)
+    {
+        $configuration = $this->requestConfigurationFactory->create($this->metadata, $request);
+
+        $order = $this->repository->findOneForPayment($orderId);
```

Comment:
This method could return null, causing a fatal error below. Should throw an exception if no order was found.